spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -275,7 +275,17 @@ object SpdxDeclaredLicenseMapping {
                 to (((licenseRef("public-domain-disclaimer", "scancode") and PYTHON_2_0.toExpression())
                 and BSD_2_CLAUSE.toExpression()) and GPL_3_0_ONLY.toExpression()),
         "the Apache License, ASL Version 2.0" to APACHE_2_0.toExpression()
-    ).let { caseSensitiveMap ->
+    ).also {
+        val keys = it.unzip().first.toMutableList()
+        val uniqueKeys = keys.distinct()
+        if (keys.size > uniqueKeys.size) {
+            uniqueKeys.forEach { uniqueKey -> keys.remove(uniqueKey) }
```

Comment:
Can't you use `keys -= uniqueKeys` or `keys.removeAll(uniqueKeys)`?

Reply:
Unfortunately not, I had tried it out: `-=` just calls `removeAll`, and `removeAll` removes *all* occurrences of a specific key from the list, which is not what we need here (we just want to remove *one* occurrence).